mypy/fastparse.py

```diff
@@ -105,7 +105,7 @@ def ast3_parse(source: Union[str, bytes], filename: str, mode: str,
         Constant = Any
 except ImportError:
     try:
-        from typed_ast import ast35  # type: ignore
+        from typed_ast import ast35  # type: ignore  # noqa: F401
```

Comment:
I considered changing this to `__import__('typed_ast').ast35` to avoid the lint error but decided to one-off exclude

Reply:
I think that (or even better `importlib.import_module`) would give mypy issues as well, so perhaps it is better to do it this way.